### PR TITLE
Bump to latest zgw-consumers

### PR DESCRIPTION
**Pull request by sergei-maertens**
This adds the django-simple-certmanager dependency, which replaces the zgw_consumers.Certificate
model.

Part one to wrap up #1849

### CODE REVIEW

**Comment by sergei-maertens** [comment]
Our migrations are a bit messed up, and so is my local database so I'm not really able to properly test these migrations.

I've added a release blocker to the 2.0.0-beta release ticket to sort out these migrations (and squash a number of them again)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,10 +2,19 @@
...
 =========
 
-1.2.0 (2022-??-??) - in development
+2.0.0 (2022-??-??) - in development
 ===================================
 
-BEFORE upgrading to 1.2.x, please read the release notes carefully.
+BEFORE upgrading to 2.0.0, please read the release notes carefully.
+
+**Upgrade procedure**
```
Comment (SilviaAmAm) [inline CHANGELOG.rst:10]: This section will be expanded in the issue https://github.com/open-formulieren/open-forms/issues/1922 ?

**Review by SilviaAmAm — APPROVED** [review]
I tried running the migrations locally and they seemed to work!

File: CHANGELOG.rst (hunk 1 above)
Comment (sergei-maertens) [reply CHANGELOG.rst:10]: #1895 rather, but yeah, this is a placeholder for now :-)

**Pull request merged**